packer/windows/conf/bin/bk-install-elastic-stack.ps1

```diff
@@ -131,6 +131,7 @@ $OFS=","
 Set-Content -Path C:\buildkite-agent\buildkite-agent.cfg -Value @"
 name="${Env:BUILDKITE_STACK_NAME}-${Env:INSTANCE_ID}-%spawn"
 token="${Env:BUILDKITE_AGENT_TOKEN}"
+endpoint="${Env:BUILDKITE_AGENT_ENDPOINT}"
```

Comment:
There are a bunch of sets above that could also include this, something like

```
set_unless_present "BUILDKITE_AGENT_ENDPOINT" "https://agent.buildkite.com/v3"
endpoint="${Env:BUILDKITE_AGENT_ENDPOINT}"
```

Again, purely for defensive defaultness, would help?

Reply:
Great suggestion, commit added 👍🏻